db/migrate/20181016140921_migrate_orch_stacks_to_have_ownership_concept.rb

```diff
@@ -0,0 +1,85 @@
+class MigrateOrchStacksToHaveOwnershipConcept < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+
+    def tenant_identity
+      User.super_admin
```

Comment:
The user may not exist, but the group does exist. I notice that you do reference `current_group` below.

```suggestion
      User.super_admin.tap { |u| u.current_group = tenant.default_miq_group }
```